share/locale/CMakeLists.txt

```diff
@@ -32,4 +33,5 @@ install(DIRECTORY ${QT_INSTALL_PREFIX}/translations/
     REGEX "qtmultimedia_.*\\.qm"
     REGEX "qtquick1_.*\\.qm"
     REGEX "qtscript_.*\\.qm"
-    REGEX "qtxmlpatterns_.*\\.qm")
+    REGEX "qtxmlpatterns_.*\\.qm"
+    REGEX "qt*_en\\.qm" EXCLUDE)
```

Comment:
Save a (tiny) bit of space by not installing the (anyway empty) en translation files